tck/pom.xml

```diff
@@ -63,21 +63,25 @@
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
+            <version>4.13.2</version>
```

Comment:
No, we should not override versions from the MP parent. That's what parent is for.

Reply:
+1, I will remove that commit from this PR
BTW the new parent does not contain junit dependency

Reply:
We might need to switch to testng but this is a problem for another time.